Formula/mingw-w64.rb

```diff
@@ -26,9 +26,9 @@ class MingwW64 < Formula
   end
 
   resource "gcc" do
-    url "https://ftp.gnu.org/gnu/gcc/gcc-8.3.0/gcc-8.3.0.tar.xz"
-    mirror "https://ftpmirror.gnu.org/gcc/gcc-8.3.0/gcc-8.3.0.tar.xz"
-    sha256 "64baadfe6cc0f4947a84cb12d7f0dfaf45bb58b7e92461639596c21e02d97d2c"
+    url "https://ftp.gnu.org/gnu/gcc/gcc-9.1.0/gcc-9.1.0.tar.gz"
```

Comment:
Does `mingw-w64` depends on this particular version of `gcc`? If not we should remove this resource and use `depends_on "gcc"` instead, unless `gcc` should be built differently for `mingw-w64`.

Reply:
It is built differently for `mingw-w64` (but I agree it would be nice to just use the "upstream" version of gcc).